CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 4.60.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_compute_region_instance_template` ([#14132](https://github.com/hashicorp/terraform-provider-google/pull/14132))
+* **New Resource:** `apigeeKeystoresAliasesKeyCertFile` ([#14130](https://github.com/hashicorp/terraform-provider-google/pull/14130))
+* **New Resource:** `google_compute_region_instance_template` ([#14103](https://github.com/hashicorp/terraform-provider-google/pull/14103))
+
+IMPROVEMENTS:
+* compute: added 3000 as allowable value for `disk_size_gb` for SCRATCH disks in `google_compute_instance_template` ([#14061](https://github.com/hashicorp/terraform-provider-google/pull/14061))
+* compute: added missing fields for `google_compute_(?.*_)address` datasource ([#14078](https://github.com/hashicorp/terraform-provider-google/pull/14078))
+* compute: added `network_firewall_policy_enforcement_order` field to `google_compute_network` resource ([#14111](https://github.com/hashicorp/terraform-provider-google/pull/14111))
+* compute: added output-only attribute `self_link_unique` id for `google_compute_instance_template` to point to the unique id of the resource (instead of its name) ([#14128](https://github.com/hashicorp/terraform-provider-google/pull/14128))
```

Comment:
```suggestion
* compute: added output-only attribute `self_link_unique` for `google_compute_instance_template` to point to the unique id of the resource instead of its name ([#14128](https://github.com/hashicorp/terraform-provider-google/pull/14128))
```

Reply:
Changed